src/cli.ts

```diff
@@ -31,7 +31,7 @@ type opt = [string, string, any];
 const mapOpt: opt = [
   "-m, --map <file>",
   "File containing initial import map",
-  { default: "importmap.json" },
+  {},
```

Comment:
This needs a documentation change though. `chomp build --force`

```
* `-m, --map` _&lt;file&gt;_                File containing initial import map 
```

But should we still give a hint to users, that the cli picks `importmap.json` if it exists ?

Reply:
Thanks, added.